### ISSUE TITLE
git clean -xfd    and now corrupted Flutter Installation. 

### ISSUE THREAD

**Issue by Andrious** [flutter/flutter#62175]
## Steps to Reproduce

1. In the attempt to fix the error:

 ***fatal: Not possible to fast-forward, aborting.*** 

I ran the [Instructions how to get Flutter installation back to a clean state](https://github.com/flutter/flutter/issues/28091)

It involved running the following git commands.
```
git clean -xfd
git stash save --keep-index
git stash drop
git pull
flutter doctor
```

2. ...I then ran flutter doctor

I suspect I should not have run those instructions in the flutter installation directory.
Looks to me the installation is now nice and damaged.

Admittedly, I have little experience with git, and don't have a clue what to do next to recover from this.
![terminal](https://user-images.githubusercontent.com/32497443/88355564-ab087480-cd2a-11ea-8c1f-f744090afdc6.jpg)
. 

## Logs

```
N/A
```

**Comment by stuartmorgan** [comment]
> I suspect I should not have run those instructions in the flutter installation directory.

That is where you should have run them. However, it looks like you had errors at one of the steps that you ignored, because the failed URL shows that you have git conflict markers it at least one file in your tree still.

You'll need to use `git status` to figure out where you still have issues, make sure all of those cases are reverted to a pristine state, and then `git pull` again.

**Comment by Andrious** [comment]
Ironically, I did run `git status` soon after posting this issue.  Again, not having enough confidence in working with git, I decided to simply set aside the 'flutter directory' (rebooted and renamed the folder) and downloaded the file, **flutter_windows_1.17.5-stable.zip**.

In other words, I simply installed Flutter again.
I'll close this issue shortly. However, I'll see if things have truly returned to normal.
Happily, I had no intention of downloading my own local repo. of Flutter.
